docs/java-troubleshooting.md

```diff
@@ -0,0 +1,31 @@
+---
+id: java-troubleshooting
+title: Scope Java Agent troubleshooting
+sidebar_label: Troubleshooting
+---
+
+## Problem
```

Comment:
Having `Problem` as the title makes the right hand side index useless:
![Screenshot 2020-02-15 at 17 25 32](https://user-images.githubusercontent.com/3635457/74591429-2ea19500-5018-11ea-87db-e31b1eddc938.png)

Instead, I'd have the problem as the title, and the solution as the body, something like:

```md
### I don't see my tests in Scope after installing the Scope Java agent

If you don't see any results in Scope after following the [Scope Java agent installation instructions](java-installation.md), check the following:

* **Set a valid `SCOPE_DSN` environment variable.** The Scope Java agent...
```